### DISCUSSION TITLE
Alternative win conditions for multiplayer/playlist lobbies

### DISCUSSION THREAD

**Discussion by snoresx3** [ppy/osu#27031] [Ideas]
This is one of the missing features that stable multiplayer lobbies have. Adding accuracy and combo as an option for a win condition is a shoe in.

For playlists:

- **Accuracy** would be a very useful alternative win condition to have as a different way to sort leaderboards for playlists with huge disparities in score multipliers between each map (e.g. if you have a playlist with DT and HT maps). 

- A weighting system based on **leaderboard position in each map** would be neat for similar reasons above.

**Comment by Rekunan** [comment]
Would also like to see pp win con

**Reply by Siiiuuuuu** [reply]
yes please, after trying out lazer mp lobbies, i saw how much different scores each player could have, 5 people playing different difficulties, 1 guy playing a different ruleset, 1 guy playing with mods.. too much difference score alone cannot handle, pp would solve this issue
